.github/workflows/notify-issue-comment.yml

```diff
@@ -0,0 +1,20 @@
+name: Issue Comment Slack Notifier
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send notification to Slack
+        uses: slackapi/slack-github-action@v1.24.0
+        with:
+          payload: |
+            {
+              "comment_url": "${{ github.event.comment.html_url }}"
```

Comment:
Should we add the title of issue for which comment was added?

Reply:
Again, the issue gets automatically unfurled and shows the start of the original issue. This shouldn't be needed